src/basho_bench_sup.erl

```diff
@@ -23,66 +23,20 @@
 
 -behaviour(supervisor).
 
-%% API
--export([start_link/0,
-         workers/0,
-         stop_child/1]).
+-export([start_link/0]).
 
-%% Supervisor callbacks
 -export([init/1]).
 
--include("basho_bench.hrl").
+-export([start_child/0]).
 
-%% Helper macro for declaring children of supervisor
--define(CHILD(I, Type), {I, {I, start_link, []}, permanent, 30000, Type, [I]}).
+-define(CHILD(I, Type), {I, {I, start_link, []}, permanent, 5000, Type, [I]}).
```

Comment:
The timeout should be an environment variable or some such. We had to bump it to 30k in 835576a9063cd92128001c748666a293219e8c69 so this would be a regression in its current form.